scala_proto/private/scalapb_aspect.bzl

```diff
@@ -230,10 +230,10 @@ scalapb_aspect = aspect(
     attrs = {
         "_protoc": attr.label(executable = True, cfg = "host", default = "@com_google_protobuf//:protoc"),
         "_implicit_compile_deps": attr.label_list(cfg = "target", default = [
-            "//external:io_bazel_rules_scala/dependency/proto/implicit_compile_deps",
```

Comment:
this is a breaking change, right?
it might be acceptable but I just want to understand the impact.
If someone doesn't use the repositories and just binds by themselves then they'll need to introduce the customization pattern, right?

Reply:
Yes, if users do not use repositories, they will have to define scalapb deps toolchain.